### PR TITLE
Add MySQL task

### PR DESCRIPTION
**Pull request by aquaruiz**
A Task for a furniture factory DB management

### CODE REVIEW

**Comment by aquaruiz** [comment]
@qnislavnachev

**Comment by aquaruiz** [comment]
Is this ok, @qnislavnachev?

File: MySQL/Scripts/Script.sql [hunk 1]
```diff
@@ -0,0 +1,277 @@
...
+VALUES 
+	('nothing happened', 12, CURRENT_TIMESTAMP());
+
+-- CREATE some Views
+
+CREATE VIEW v_get_products_data AS
+	SELECT p.id, p.name, p.shortDescription, pd.weight, pd.price, pd.barcodeNumber 
+		FROM product AS p
+		JOIN productDetail AS pd
+		ON p.id = pd.id;
```
Comment (qnislavnachev) [inline MySQL/Scripts/Script.sql]: p.id = pd.productId

File: MySQL/Scripts/Script.sql [hunk 2]
```diff
@@ -0,0 +1,277 @@
...
+
+SELECT * FROM v_get_products_data; 
+
+CREATE VIEW v_get_customers_data AS
+	SELECT c.id, cd.name, c.financialId, c.vatNumber, cd.address, cd.contactPerson 
+		FROM customer AS c 
+		JOIN customerDetail AS cd 
+		ON c.id = cd.id;
```
Comment (qnislavnachev) [inline MySQL/Scripts/Script.sql]: c.id = cd.customerId

File: MySQL/Scripts/Script.sql [hunk 3]
```diff
@@ -0,0 +1,277 @@
...
+		JOIN product AS p
+		ON p.id = po.productId 
+		JOIN productDetail AS pd
+		ON p.id = pd.id 
+		JOIN customerOrder AS o
+		ON o.id = po.orderId 
+		JOIN customer AS c
+		ON c.id = o.customerId
+		JOIN customerDetail AS cd
+		ON cd.id  = c.id
```
Comment (qnislavnachev) [inline MySQL/Scripts/Script.sql]: c.id = cd.customerId

File: MySQL/Scripts/Script.sql [hunk 4]
```diff
@@ -0,0 +1,277 @@
...
+(
+   orderNumber INT
+) 
+BEGIN  
+	SELECT po.orderId, c.vatNumber, cd.name, p.name, po.productQuantity, pd.price AS 'price per unit', pd.price * po.productQuantity AS 'total price' 
+		FROM productOrder AS po
+		JOIN product AS p
+		ON p.id = po.productId 
+		JOIN productDetail AS pd
+		ON p.id = pd.id
```
Comment (qnislavnachev) [inline MySQL/Scripts/Script.sql]: p.id = pd.productId

File: MySQL/Scripts/Script.sql [hunk 5]
```diff
@@ -0,0 +1,277 @@
...
+
+CREATE PROCEDURE sp_get_orders_for_last_month_by_customer_name
+(
+   customerName VARCHAR(255)
+) 
+BEGIN
+	SELECT o.id AS 'ORDER id', o.totalPrice, o.orderDate, cd.name AS 'customer name'
+		FROM customerOrder AS o 
+		JOIN customerDetail AS cd
+		ON cd.id = o.customerId
```
Comment (qnislavnachev) [inline MySQL/Scripts/Script.sql:191]: cd.customerId = o.customerId

Check all places for such error

File: MySQL/Scripts/Script.sql [hunk 6]
```diff
@@ -0,0 +1,277 @@
...
+
+-- DROP view sp_get_products_sold_last_month;
+
+-- add indices
+
+CREATE INDEX index_product
+ON productDetail (id);
+
+CREATE INDEX indexOrder
+ON productOrder (orderId, productId);
```
Comment (qnislavnachev) [inline MySQL/Scripts/Script.sql:223]: I dont think you need this indexes. I think there is better columns to be indexed here.

**Review by qnislavnachev — CHANGES_REQUESTED** [review]

**Pull request merged**